configure.ac

```diff
@@ -931,7 +931,7 @@ OPAL_CHECK_BROKEN_QSORT
 #
 # Check out what thread support we have
 #
-OPAL_CONFIG_THREADS
+#OPAL_CONFIG_THREADS
```

Comment:
we need to set these compiler flags now in the threads framework in the various components, we may need to do something special here